guides/source/form_helpers.md

```diff
@@ -64,184 +64,227 @@ To create this form you will use `form_with` and the form builder object it yiel
 This will generate the following HTML:
 
 ```html
-<form action="/search" method="get" accept-charset="UTF-8" >
+<form action="/search" accept-charset="UTF-8" method="get">
   <label for="query">Search for:</label>
-  <input id="query" name="query" type="text" />
-  <input name="commit" type="submit" value="Search" data-disable-with="Search" />
+  <input type="text" name="query" id="query">
+  <input type="submit" name="commit" value="Search" data-disable-with="Search">
 </form>
 ```
 
-TIP: Passing `url: my_specified_path` to `form_with` tells the form where to make the request. However, as explained below, you can also pass Active Record objects to the form.
+Notice that for the search form we are using the `url` option of `form_with`. Setting `url: "/search"` changes the form action value from the default current page path to `action="/search"`.
 
-TIP: For every form input, an ID attribute is generated from its name (`"query"` in above example). These IDs can be very useful for CSS styling or manipulation of form controls with JavaScript.
+In general, passing `url: my_path` to `form_with` tells the form where to make the request. The other option is to pass Active Record objects to the form, as you will learn [below](#creating-forms-with-model-objects).
 
-IMPORTANT: Use "GET" as the method for search forms. This allows users to bookmark a specific search and get back to it. More generally Rails encourages you to use the right HTTP verb for an action.
+The search form example also shows the [form builder](https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html) object. You will learn about the many helpers provided by the form builder object (like`form.label` and `form.text_field` in the above example) in the next section.
+
+TIP: For every form `input` element, an `id` attribute is generated from its name (`"query"` in above example). These IDs can be very useful for CSS styling or manipulation of form controls with JavaScript.
+
+IMPORTANT: It is recommended to use "GET" as the method for search forms. This allows users to bookmark a specific search. More generally Rails encourages you to use the right HTTP verb for an action.
```

Comment:
"More generally Rails encourages you to use the right HTTP verb for an action."
Does this mean that you should use GET for search forms because its the right HTTP verb to use? Seems unnecessary unless leading with that:
"Rails convention encourages using the right HTTP verb for an action, so use "GET" as the method for search forms. This allows users to bookmark and return to a specific search."